conflicts.lua

```diff
@@ -1925,29 +1925,36 @@ local t = {
+	["HasAction"] = [=[Returns 1 if ActionBar [[ActionSlot]] is assigned an action.]=],
 
-	["HasPetSpells"] = [=[Returns true if the player has pet spells.]=],
-	["HasPetSpells"] = [=[Returns how many abilities your pet has available.]=],
+	-- ["HasPetSpells"] = [=[Returns true if the player has pet spells.]=],
+	-- ["HasPetSpells"] = [=[Returns how many abilities your pet has available.]=],
+	["HasPetSpells"] = [=[True if the player has pet spells.]=],
 
-	["HasPetUI"] = [=[Returns 1 if the player has a pet User Interface.]=],
-	["HasPetUI"] = [=[Returns <code>True</code> if the player has a pet User Interface.]=],
+	-- ["HasPetUI"] = [=[Returns 1 if the player has a pet User Interface.]=],
+	-- ["HasPetUI"] = [=[Returns <code>True</code> if the player has a pet User Interface.]=],
+	["HasPetUI"] = [=[True if the player has a pet interface.]=],
 
-	["HasWandEquipped"] = [=[Returns 1 if a wand is equipped, false otherwise.]=],
-	["HasWandEquipped"] = [=[HasWandEquipped();]=],
+	-- ["HasWandEquipped"] = [=[Returns 1 if a wand is equipped, false otherwise.]=],
+	-- ["HasWandEquipped"] = [=[HasWandEquipped();]=],
+	["HasWandEquipped"] = [=[True if a wand is equipped.]=],
 
-	["HideRepairCursor"] = [=[Hides the repair cursor.]=],
+	-- ["HideRepairCursor"] = [=[Hides the repair cursor.]=],
 	["HideRepairCursor"] = [=[Takes the cursor out of repair mode.]=],
 
-	["InCombatLockdown"] = [=[Returns true if the in-combat AddOn restrictions are active.]=],
-	["InCombatLockdown"] = [=[Determines whether in-combat lockdown restrictions are active.]=],
+	-- ["InCombatLockdown"] = [=[Returns true if the in-combat AddOn restrictions are active.]=],
+	-- ["InCombatLockdown"] = [=[Determines whether in-combat lockdown restrictions are active.]=],
+	["InCombatLockdown"] = [=[True if the in-combat AddOn restrictions are active.]=],
 
-	["InRepairMode"] = [=[Returns true if your cursor is in repair mode.]=],
-	["InRepairMode"] = [=[Lets you know if your cursor is in repair mode. When your cursor is in repair mode, you can click on equipped items as well as items in your inventory to repair them.]=],
+	-- ["InRepairMode"] = [=[Returns true if your cursor is in repair mode.]=],
+	-- ["InRepairMode"] = [=[Lets you know if your cursor is in repair mode. When your cursor is in repair mode, you can click on equipped items as well as items in your inventory to repair them.]=],
+	["InRepairMode"] = [=[True if cursor is in repair mode.]=],
 
-	["InboxItemCanDelete"] = [=[Boolean check for whether a message is returnable to the sender.]=],
-	["InboxItemCanDelete"] = [=[Boolean function for determining whether a message is returnable.]=],
+	-- ["InboxItemCanDelete"] = [=[Boolean check for whether a message is returnable to the sender.]=],
+	-- ["InboxItemCanDelete"] = [=[Boolean function for determining whether a message is returnable.]=],
+	["InboxItemCanDelete"] = [=[False if a mailed item or money is returnable, true otherwise.]=],
```

Comment:
Not sure about this one. I thought the Return description explained what it does better than the other 2 https://wowpedia.fandom.com/wiki/API_InboxItemCanDelete Figured toss it in here let you make the final call